ci/pkl-pipelines/pay-deploy/pay-js.pkl

```diff
@@ -11,11 +11,21 @@ local class PayJsLibraryPipeline {
 
 local payJsLibraries: Listing<PayJsLibraryPipeline> = new {
   new {
-    name = "commons"
+    name = "pay-js-commons"
```

Comment:
These are instances of the classes on line 7, so you need to add the new properties to that class.

One of the github actions is failing, if you click on the link for that it should take you to the concourse job that failed, opening the failed task will show you the error.

You can also run it yourself and see them locally.


If you `brew install pkl` you can then run in `pay-ci/ci/pkl-pipelines`

```
pkl eval pay-deploy/pay-js.pkl
```

Reply:
Thanks, I'll check that out :)

Reply:
@jfharden (and anyone else who's got an opinion)

Are there any sensible semantic names for the different variations of the name?

Before this change the name was just either `commons` or `metrics` which is extended out automatically to `js-commons` and `pay-js-commons` but that's not appropriate for `run-amock`.

I'd be tempted to call them:

- `full_name` (e.g. `pay-js-commons`, `pay-js-metrics`, `run-amock`)
- `short_name` (e.g. `commons`, `metrics`, `run-amock`)
- `internal_name` (e.g. `js-commons`, `js-metrics`, `run-amock`)

Does that sound vaguely sensible?  Any other/better/different suggestions?